package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pushwoosh-cordova-plugin",
-  "version": "7.8.3-OS3",
+  "version": "7.11.0-OS3",
```

Comment:
Version suffix should be reset to `7.11.0-OS` given that there's still custom code to fit OutSystems needs

Reply:
Indeed. Let that one slip past me...

Reply:
Thanks for the heads up, it's fixed now.